code/game/gamemodes/cult/ritual.dm

```diff
@@ -165,522 +10,268 @@ var/engwords = list("travel", "blood", "join", "hell", "destroy", "technology",
-					R.check_icon()
-				if("bloodboil")
-					R.word1=cultwords["destroy"]
-					R.word2=cultwords["see"]
-					R.word3=cultwords["blood"]
-					R.loc = user.loc
-					R.check_icon()
-				if("communicate")
-					R.word1=cultwords["self"]
-					R.word2=cultwords["other"]
-					R.word3=cultwords["technology"]
-					R.loc = user.loc
-					R.check_icon()
-				if("stun")
-					R.word1=cultwords["join"]
-					R.word2=cultwords["hide"]
-					R.word3=cultwords["technology"]
-					R.loc = user.loc
-					R.check_icon()
+				chosen_keyword = the_keyword
+			break
+	if(!rune_to_scribe)
+		return
+	Turf = get_turf(user) //we may have moved. adjust as needed...
+	if(locate(/obj/effect/rune) in Turf)
+		to_chat(user, "<span class='cult'>There is already a rune here.</span>")
+		return
+	if(!Adjacent(user) || !src || qdeleted(src) || user.incapacitated())
+		return
+	if(ispath(rune_to_scribe, /obj/effect/rune/narsie) || ispath(rune_to_scribe, /obj/effect/rune/slaughter))//may need to change this - Fethas
+
+		if(ticker.mode.name == "cult")
+			var/datum/game_mode/cult/cult_mode = ticker.mode
+			if(!("eldergod" in cult_mode.objectives))
+				to_chat(user, "<span class='warning'>[cult_mode.cultdat.entity_name]'s power does not wish to be unleashed!</span>")
+				return
+			else if(!cult_mode.narsie_condition_cleared)
+				to_chat(user, "<span class='warning'>There is still more to do before unleashing [cult_mode.cultdat.entity_name] power!</span>")
+				return
+			else if(!cult_mode.eldergod)
+				to_chat(user, "<span class='cultlarge'>\"I am already here. There is no need to try to summon me now.\"</span>")
+				return
+			else if(cult_mode.demons_summoned)
+				to_chat(user, "<span class='cultlarge'>\"We am already here. There is no need to try to summon us now.\"</span>")
+				return
+			var/confirm_final = alert(user, "This is the FINAL step to summon Nar-Sie, it is a long, painful ritual and the crew will be alerted to your presence", "Are you prepared for the final battle?", "My life for Nar-Sie!", "No")
+			if(confirm_final == "No")
+				user << "<span class='cult'>You decide to prepare further before scribing the rune.</span>"
+				return
+			command_announcement.Announce("Figments from an eldritch god are being summoned somwhere on the station from an unknown dimension. Disrupt the ritual at all costs!","Central Command Higher Dimensionsal Affairs", 'sound/AI/spanomalies.ogg')
+			for(var/B in spiral_range_turfs(1, user, 1))
+				var/turf/T = B
+				var/obj/machinery/shield/N = new(T)
+				N.name = "Rune-Scriber's Shield"
+				N.desc = "A potent shield summoned by cultists to protect them while they prepare the final ritual"
+				N.icon_state = "shield-red"
+				N.health = 60
+				shields |= N
+	var/dam_zone = pick("head", "chest", "groin", "l_arm", "l_hand", "r_arm", "r_hand", "l_leg", "l_foot", "r_leg", "r_foot")
+	user.visible_message("<span class='warning'>[user] cuts open their [dam_zone] and begins writing in their own blood!</span>", \

```

Comment:
This might be better:

`var/dam_zone = pick("head", "chest", "groin", "l_arm", "l_hand", "r_arm", "r_hand", "l_leg", "l_foot", "r_leg", "r_foot")`
`var/obj/item/organ/external/affecting = get_organ(ran_zone(dam_zone))`
`user.visible_message("<span class='warning'>[user] cuts open their [affecting] and begins writing in their own blood!</span>", "<span class='cult'>You slice open your [affecting] and begin drawing a sigil of [ticker.mode.cultdat.entity_title3].</span>")`
`user.apply_damage(initial(rune_to_scribe.scribe_damage), BRUTE, affecting)`